Source/Renderer/ShaderProgram.js

```diff
@@ -16,7 +16,18 @@ import createUniformArray from './createUniformArray.js';
      * @private
      */
     function ShaderProgram(options) {
-        var modifiedFS = handleUniformPrecisionMismatches(options.vertexShaderText, options.fragmentShaderText);
+        var vertexShaderText = options.vertexShaderText;
+        var fragmentShaderText = options.fragmentShaderText;
+
+        if (window.spector) {
```

Comment:
While we don't do a good job of it everywhere, in general this should be `typeof spector !== 'undefined'` instead using `window`, since `window` doesn't exist in all contexts (workers/node/etc..) it's a more robust check for a global `spector` variable

Reply:
Done!